backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/Einzelelement.java

```diff
@@ -0,0 +1,34 @@
+package de.bund.digitalservice.ris.norms.domain.entity;
+
+import lombok.Builder;
+import lombok.Getter;
+import org.w3c.dom.Node;
+
+/** Class representing the meta:legalDocML.de_metadaten_ds/meta:einzelelement */
+@Getter
+public class Einzelelement extends Metadaten<Einzelelement.Metadata> {
+
+  @Builder
+  public Einzelelement(final Node node) {
+    super(node, "start", "end");
+  }
+
+  /**
+   * The list of all simple metadata within the meta:legalDocML.de_metadaten_ds block. They consist
+   * of a single string property.
+   */
+  public enum Metadata implements MetadataInterface {
+    ART_DER_NORM("./artDerNorm");
```

Comment:
Should we maybe create a general enum with metadata (and xpath) present in several metadata blocks, like for example the FNA in MetadatenDe and MetadatenDs and artDerNorm in both MetadatenDs and Einzelelemente? Or you think is more readable to duplicate the metadata definition with their xpaths in every metadata block?

Reply:
I don't get this one. We might need to discuss.